.circleci/config.yml

```diff
@@ -42,6 +47,10 @@ aliases:
     name: Create database
     command: cd src/api; bundle exec rake db:setup RAILS_ENV=test
 
+  - &create_test_db-next_rails
+    name: Create database based on gems defined in src/api/Gemfile.next
```

Comment:
Create database based on gems? huh?

Reply:
To be honest... I don't know how to phrase this. I create the database and the next Rails version might changed how this is done. Let's say the rake tasks changed in the next Rails version, the command would have to be changed. This would probably not happen, but I still want to run with `next bundle exec rake (...)`.

Reply:
`Create database based with next rails version`